examples/thalamus_sample_dir/LICENSE.txt

```diff
@@ -0,0 +1,37 @@
+The CC-BY-NC-SA license applies, as indicated by headers in the
+respective source files.
+
+https://creativecommons.org/licenses/by-nc-sa/4.0/
+
+The detailed text is available here
+https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode
+
+or in this tarball in the seperate file LICENSE_CC-BY-CA-SA-4.0
+
+The HOC code, Python code, synapse MOD code and cell morphology are licensed with the above mentioned CC-BY-NC-SA license.
+
+For models for which the original source is available on ModelDB, any
+specific licenses on mentioned on ModelDB, or the generic License of ModelDB
+apply:
+
```

Comment:
Maybe you should add to the following list of mechanisms all the thalamus-specific mechanisms that are not present yet

Reply:
Nice catch. I need @elisabettai's input on this. Let me create an issue and refer to it on the ticket.

Reply:
#35